code/modules/surgery/organs/organ_internal.dm

```diff
@@ -15,6 +15,11 @@
 	if(istype(holder))
 		insert(holder)
 
+/obj/item/organ/internal/Destroy()
```

Comment:
Remove this entire `Destroy`: https://github.com/ParadiseSS13/Paradise/blob/4a4cdba6ac4202fe3e7b80b1af9cd3f6b645d2b2/code/modules/surgery/organs/organ.dm#L35